docs/docs/basics/testing.md

```diff
@@ -4,9 +4,27 @@ sidebar_position: 5
 
 # Testing 🧪
 
-In Dart Frog, we can unit test our route handlers and middleware effectively because they are plain functions.
+In Dart Frog, we can effectively unit test our route handlers and middleware using [`package:test`](https://pub.dev/packages/test) and [`package:mocktail`](https://pub.dev/packages/mocktail).
 
-For example, we can test our route handler above using `package:test`:
+## Route Handlers 🚏
+
+Testing route handlers is pretty straightforward and doesn't require any new concepts because at the end of the day a route handler is just a plain Dart function.
```

Comment:
```suggestion
Testing route handlers is pretty straightforward and doesn't require any new concepts because a route handler is just a plain Dart function.
```